truenas/webui
# NAS-111620 / 21.08 / Fix app-bootenv-rename

Testing: 
- Go to the boot environment rename form (system/boot/rename/foo)
- Make sure form actually renders the input field
- Input field should display placeholder text
- Make sure name is successfully updated after submit
